SPECS/xapi-storage.spec

```diff
@@ -103,13 +103,14 @@ cp storage/api/__init__.py storage/api/datapath.py storage/api/volume.py storage
 
 %files ocaml-plugin-devel
 %defattr(-,root,root,-)
+%exclude %{_libdir}/ocaml/xapi-storage/*.ml
```

Comment:
What does it actually do, because the same line appears 5 lines down, but without `%exclude`?